LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2021 FIXME
```

Comment:
We should figure out who to put here

Reply:
I believe it should be Tom as the owner of the original project. My understanding of it is that the original author license's the work. If you modify the code you retain your right to your changes but when you push back to the original repo you give the author rights to distribute your changes under the original license. I'm not a lawyer so but that's how I always took it. I imagine in a real dispute you'd argue over if you implicitly gave over your permission with the PR or if an additional agreement was needed but this is a pretty small project with a very permissive license so hopefully we're all cool. 

I checked a few other projects and most people don't even bother to change those lines which was kind of funny.

Reply:
@TomWhitwell are you cool with that?

Reply:
Yes I'm cool with either no name, my name or lots of names from contributors.

Reply:
Resolved by 6989241